apis/core/v1beta1/common/common.go

```diff
@@ -12,6 +12,36 @@ const (
 	SyncProviderFlagdProxy SyncProviderType = "flagd-proxy"
 )
 
+const (
+	MetricPortEnvVar            string = "MANAGEMENT_PORT"
```

Comment:
to keep it consistent I think we could also rename this to `ManagementPortEnvVar`:

```suggestion
	ManagementPortEnvVar            string = "MANAGEMENT_PORT"
```